GraphicsForAxisEvents/src/main.ts

```diff
@@ -0,0 +1,109 @@
+import * as fs from "fs";
+import { CameraVapix } from "camstreamerlib/CameraVapix";
+import { CamOverlayAPI } from "camstreamerlib/CamOverlayAPI";
+
+type Camera = {
+  IP: string;
+  port: number;
+  user: string;
+  password: string;
+};
+
+type Settings = {
+  sourceCamera: Camera;
+  targetCamera: Camera;
+  modem: Camera;
+  events: {
+    eventName: string;
+    serviceID: number;
+    duration: number;
+  }[];
+};
+
+type Event = {
+  eventName: string;
+  duration: number;
+  co: CamOverlayAPI;
+};
+
+let cv: CameraVapix;
```

Comment:
kdyz to malinko zrefaktorujes tak "cv" vubec nemusi byt globalni a bude kod i lepe citelny..

Reply:
A settings refaktorovat nemám? Mám pocit, že jsou na tom úplně stejně jako cv.